[Linux][1.11.0] Blueprint does not start: "cannot find fix-path'"

<!-- Please provide a detailed description of the bug. -->
### Bug Description: 

Blueprint v1.11.0 does not start. I get the same error with the `.Appimage` or the `Debian installer` version:
$ /opt/TheiaBlueprint/theia-blueprint 
```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'fix-path'
Require stack:
- /opt/TheiaBlueprint/resources/app/src-gen/frontend/electron-main.js
- /opt/TheiaBlueprint/resources/app/scripts/theia-electron-main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:797:17)
    at Function.o._resolveFilename (electron/js2c/browser_init.js:281:679)
    at Module._load (internal/modules/cjs/loader.js:690:27)
    at Function.Module._load (electron/js2c/asar.js:769:28)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/opt/TheiaBlueprint/resources/app/src-gen/frontend/electron-main.js:9:1)
    at Module._compile (internal/modules/cjs/loader.js:967:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1004:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
```

<!-- Please provide clear steps to reproduce the bug. -->
### Steps to Reproduce:

1. download .deb or AppImage  from https://download.eclipse.org/theia/1.11.0/linux/)
2. chmod +x and start .AppImage or for .deb start: `/opt/TheiaBlueprint/theia-blueprint`
3.

<!-- Please provide any additional information available. -->
<!-- Additional information can be in the form of logs, screenshots, screencasts. -->

### Additional Information

- Operating System: Ubuntu 20.04
- Theia Version: 1.11

I have not tested the Windows or MacOS packages, so I assume the issue may be Linux-specific.

--- a/electron-app/package.json
+++ b/electron-app/package.json
@@ -39,6 +39,7 @@
     "@theia/debug": "1.11.0",
     "@theia/editor": "1.11.0",
     "@theia/editor-preview": "1.11.0",
+    "@theia/electron": "1.11.0",    
     "@theia/file-search": "1.11.0",
     "@theia/filesystem": "1.11.0",
     "@theia/getting-started": "1.11.0",
@@ -75,7 +76,6 @@
   },
   "devDependencies": {
     "@theia/cli": "1.11.0",
-    "@theia/electron": "1.11.0",
     "@wdio/cli": "^6.10.2",
     "@wdio/local-runner": "^6.10.2",
     "@wdio/mocha-framework": "^6.8.0",

